### PR TITLE
Revert "hooks: mv docker user/group definition to extrausers"

### PR DESCRIPTION
**Pull request by xnox**
This reverts commit 224a6c62c5518eb8b26aa69b22063b5bd04e64d0.

### CODE REVIEW

**Comment by anonymouse64** [comment]
This is ok to go back to the status quo, but as per the meeting yesterday, we also want to drop the docker group from `/etc/{group,passwd}` entirely so there are no references to a docker group at all anywhere on a UC20 system, so while in the right direction I don't think this is sufficient

**Review by anonymouse64 — APPROVED** [review]
lgtm, thanks for adding the additional checks

File: hooks/990-ensure-uids-gids.chroot [hunk 1]
```diff
@@ -20,20 +20,16 @@ echo "Ensure gshadow file does not change"
 diff -u /etc/gshadow /etc/gshadow.orig
 rc=$?; [ "$rc" != "0" ] && MISMATCH=1
 
-echo "Ensure extrausers passwd file does not change"
-diff -u /var/lib/extrausers/passwd /var/lib/extrausers/passwd.orig
+echo "Ensure no docker"
+! grep -q docker /etc/passwd /etc/shadown /etc/group /etc/gshadow
```
Comment (anonymouse64) [inline hooks/990-ensure-uids-gids.chroot:24]: mmm travis seems unhappy about this line

Comment (xnox) [reply hooks/990-ensure-uids-gids.chroot]: works now.

**Pull request merged**